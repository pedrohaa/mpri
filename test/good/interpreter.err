File "good/interpreter.ff", line 99, characters 8-12:
Type mismatch.
Expected: term (cons e domain) codomain
Inferred: term (cons _ domain) codomain
Hypotheses:
domain = alpha1
codomain = alpha2
