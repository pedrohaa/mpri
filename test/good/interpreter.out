forall a. arrow a a
